src/api.js

```
@@ -1,40 +1,31 @@
 
-// note: api base url set in package.json proxy
-
-// start with separate GET and POST calls with minimal error handling, 
-// I can make it pretty later
-
-const apiGet = async (route) => {
-    return (fetch(`${route}`, {
-      method: 'GET',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-    })
-      .then((res) => res.json())
-      .then((json) => {  
-        console.log({ apiGetResults: json });
-        return json;
-      })
-      .catch((err) => console.log(err)))
```

Don't catch errors if you don't handle them properly. Let them propagate.